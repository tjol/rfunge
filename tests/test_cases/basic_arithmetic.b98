aa+7-:.v>0/.de+5v
:,:.:*5<^./3;@.%<;