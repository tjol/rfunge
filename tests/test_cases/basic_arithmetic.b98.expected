13 65 A21 0 2 